tracardi/process_engine/action/v1/operations/contains_pattern/plugin.py

```diff
@@ -0,0 +1,152 @@
+from pydantic import validator
+import re
+
+from tracardi.process_engine.action.v1.operations.contains_pattern.patterns import patterns
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.config import PluginConfig
+from tracardi.service.plugin.domain.result import Result
+
+
+class Config(PluginConfig):
+    field: str
+    pattern: str
+
+    @validator("field")
+    def validate_prefix(cls, value):
+        if value == "":
+            raise ValueError("Field cannot be empty")
+        return value
+
+    @validator("pattern")
+    def validate_field(cls, value):
+        if value == "":
+            raise ValueError("Pattern cannot be empty")
+        return value
+
+
+def validate(config: dict) -> Config:
+    return Config(**config)
+
+
+class WrongFieldTypeError(Exception):
+    """Raised when given field has wrong data type"""
+    pass
+
+
+class EmptyPatternError(Exception):
+    """Raised when pattern field is empty"""
+    pass
+
+
+class ContainsPatternAction(ActionRunner):
+    config: Config
+
+    async def set_up(self, init):
+        self.config = validate(init)
+
+    async def run(self, payload: dict, in_edge=None):
+        dot = self._get_dot_accessor(payload)
+        value = dot[self.config.field]
+
+        if not isinstance(value, str) or isinstance(value, list):
```

Comment:
if not (isintance(value, str) or isintance(value, list)). For now, List will raise an error that is unwanted.